server/package.json

```diff
@@ -15,7 +15,7 @@
 
     "test": "yarn test:api",
     "test:api": "yarn test:dredd; yarn dev:test:kill",
-    "test:dredd": "babel-node node_modules/.bin/dredd",
+    "test:dredd": "NODE_ENV=test babel-node node_modules/.bin/dredd",
```

Comment:
@AleksSenkou shouldn't we use `NODE_ENV=test` for every test related command?

Reply:
I mean we need to add separate command which will setup env variables for every environment from `env.test` or `env.development` files, or smth similar to this

Reply:
> shouldn't we use NODE_ENV=test for every test related command?

@DDKatch we should if it actual call, not a chain of tasks. So, for now only `test:dredd` needs it

Reply:
> I think we need to add separate command which will setup env variables for every environment from env.test or env.development files, or smth similar to this

Can you give more notes about it?